src/v0/util/index.js

```diff
@@ -2039,7 +2039,14 @@ const getAuthErrCategoryFromStCode = (status) => {
   }
   return '';
 };
-
+const isValidInteger = (value) => {
+  if (Number.isNaN(value) || !isDefinedAndNotNull(value)) {
+    return false;
+  }
+  // const valueToString = `${value}`;
+  // Use a regular expression to check if the string is a valid integer or a valid floating-point number
+  return /^-?\d+(\.\d+)?$/.test(`${value}`);
```

Comment:
should we try to do this after typeof value === 'string'

Reply:
done